contents/docs/data-model.mdx

```diff
@@ -1,38 +0,0 @@
----
-title: Data model
-sidebar: Docs
-showTitle: true
----
-
-
-## Event (`posthog_event`)
-
-
-| Property | Type | Description |
-| --- | --- | --- |
-| **team_id** | ForeignKey(Team) | Link to the team |
-| **event** | CharField | Name of the event.<br />There are default events that have special meaning within PostHog:<br />- `$pageview` captures a pageview<br />- `$autocapture` anything that was captured automatically by posthog-js. Includes clicks, changes to input and submissions of forms. Probably requires Elements to be created.<br />- `$identify`, which is used in the backend to set Person properties<br />- `$create_alias`, which creates an alias between a previous distinct_id and the current one |
-| **distinct_id** | CharField | The unique or anonymous id of the user that triggered the event.<br />NOTE: migration `0024` adds indexes to this, which is necessary for /paths to work reasonably quickly |
-| **properties** | JSONField | Any key: value pairs in a dict.<br />- `$current_url` - we use this in a couple of places (like /paths, /events) as the url the user was visiting at that time. |
-| **timestamp** | DateTimeField | Defaults to timezone.now if not set |
-| **ip** | GenericIPAddressField | IP of the user |
-| **elements** | JSONField | Deprecated in favour of the Element model |
-
-
-## Element
```

Comment:
we should keep this part & I quite like the table format of this, just need to add person_id and person_properties - knowing the exact field names is useful for self-hosted folks

Reply:
Got it! I'll merge this with the current data model page and add tables for the `Person` and `Event` types. Eventually it'd be awesome to have all of our tables in here so I can work on that in the future.